lib/commands/project/project.js

```diff
@@ -479,3 +498,80 @@ function installDependencies(directory, message, cb) {
     cb(null, message);
   });
 }
+
+function writeTestFiles(directory, tests, conflicted, overwrite, callback) {
+  var index;
+  var file;
+
+  if (overwrite === OVERWRITES[2]) { // prompt on conflicted files
+    var confirmations = [];
+
+    // queues conflicted files for confirmation and
+    // writes non-conflicted files
+    async.each(tests, function (item, callback) {
+      if (conflicted.indexOf(item.name) !== -1) {
+        var temp = function(callback) {
+          var message = 'Overwrite ' + item.name + '?\n';
+          var question = { name: 'x', message: message, type: 'confirm'};
+
+          inquirer.prompt(question, function(answers) {
+            callback(null, {data: item, ans: answers.x});
+          });
+        }
+
+        confirmations.push(temp);
+      } else {
+        fs.outputFile(path.join(directory, '/test', item.name), item.test,
+          callback);
+      }
+    }, callback);
+
+    // runs confirmations of conflicted files and
+    // writes those that are confirmed for overwriting
+    async.series(confirmations, function(err, result) {
+      var ndx;
+      var overwriteCount = 0;
+
+      for (ndx in result) {
+        if (result[ndx].ans) {
+          overwriteCount++;
+          fs.outputFile(path.join(directory, '/test', result[ndx].data.name),
+            result[ndx].data.test);
+        }
+      }
+
+      console.log('done overwriting ' + overwriteCount + ' file(s).');
+    });
+  } else if(overwrite === OVERWRITES[1]) { // overwrite all conflicted files
+    async.each(tests, function (item, callback) {
+      fs.outputFile(path.join(directory, '/test', item.name), item.test,
+        callback);
+    }, callback);
+  } else { // do not overwrite any conflicted file
+    async.each(tests, function (item, callback) {
+      if (conflicted.indexOf(item.name) === -1) {
```

Comment:
Instead of repeating `async.each` block you might be able to make the code simpler and shorter by putting `(overwrite === OVERWRITES[1])` in here and do `async.each` once